web/client/epics/playback.js

```diff
@@ -30,7 +32,13 @@ const BUFFER_SIZE = 20;
 const PRELOAD_BEFORE = 10;
 const domainArgs = (getState, paginationOptions = {}) => {
     // const timeData = timeDataSelector(getState()) || {};
-    return ['http://cloudsdi.geo-solutions.it:80/geoserver/gwc/service/wmts', "landsat8:B3", "time", {
+    const selectedLayer = selectedLayerSelector(getState());
```

Comment:
```suggestion
   const selectedLayerId = selectedLayerSelector(getState());
    const layer = get(getLayerFromId(getState(), selectedLayerId)
    const layerName = layer && layer.name;
    const layerUrl = layer && layers.dimensions && layer.dimensions.filter(...)
```
This to avoid possible exceptions